x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -148,13 +167,27 @@ export const ConditionEntryInput = memo<ConditionEntryInputProps>(
               data-test-subj={getTestId('field')}
             />
           </ConditionEntryCell>
-        </EuiFlexItem>
-        <EuiFlexItem>
+        </InputItem>
+        <InputItem style={{ gridArea: 'operator' }}>
           <ConditionEntryCell showLabel={showLabels} label={ENTRY_PROPERTY_TITLES.operator}>
-            <EuiFieldText name="operator" value={OPERATOR_TITLE.included} readOnly />
+            {entry.field === ConditionEntryField.PATH ? (
+              <EuiSuperSelect
+                options={operatorOptions}
+                onChange={handleOperatorUpdate}
+                valueOfSelected={entry.type}
+                data-test-subj={getTestId('operator')}
+              />
+            ) : (
+              <EuiFieldText
+                name="operator"
+                value={OPERATOR_TITLES.is}
+                data-test-subj={getTestId('operator')}
+                readOnly
+              />
+            )}
           </ConditionEntryCell>
-        </EuiFlexItem>
-        <EuiFlexItem grow={3}>
+        </InputItem>
+        <InputItem style={{ gridArea: 'value' }}>
           <ConditionEntryCell showLabel={showLabels} label={ENTRY_PROPERTY_TITLES.value}>
             <EuiFieldText
               name="value"
```

Comment:
This is mainly a UI observation and can be addressed in a follow-up PR but with the addition of the `matches` operator, and the similarities it has with the `is` operator at least from a language perspective, it might be nice to show two different placeholders in the `Value` box to help users with less context differentiate between the two. Perhaps switching back and forth between `C://sample/path.exe` and `C://sample/**/*` or something akin to that

Reply:
Good observation @dplumlee. I shall update the placeholder.

Reply:
done 6f2d0d78104673a4b18a6c0d04a93842ffaf06a5